### PR TITLE
moyo-header second PR

### PR DESCRIPTION
**Pull request by postrygan**
https://postrygan.github.io/layout_moyo-header/
https://postrygan.github.io/layout_moyo-header/report/html_report/

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,74 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", Arial, sans-serif;
+}
+
+body {
+  margin: 0;
+  background-color: #e5e5e5;
```
Comment (tetiana-ivanchenko) [inline src/style.css:7]: redundant

File: src/style.css [hunk 2]
```diff
@@ -1 +1,74 @@
...
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  background-color: #fff;
```
Comment (tetiana-ivanchenko) [inline src/style.css:13]: redundant

File: src/style.css [hunk 3]
```diff
@@ -1 +1,74 @@
...
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.header_item {
+  display: flex;
+  justify-content: center;
+
+}
```
Comment (tetiana-ivanchenko) [inline src/style.css:39]:
```suggestion
  justify-content: center;
}
```

File: src/style.css [hunk 4]
```diff
@@ -1 +1,74 @@
...
+
+.header_item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.header_link {
+  position: relative;
+  color: #000;
+  font-family: inherit;
```
Comment (tetiana-ivanchenko) [inline src/style.css:48]: redundant

File: src/style.css [hunk 5]
```diff
@@ -1 +1,74 @@
...
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14px;
+  text-decoration: none;
+  text-transform: uppercase;
+  padding: 23px 0;
+}
+
+.is-active {
+  color: #00acdc;
```
Comment (tetiana-ivanchenko) [inline src/style.css:58]: move repeatable colors into the variable

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 2 above)
Comment (postrygan) [reply src/style.css:13]: Cannot remove dispaly:flex; and justify-content: space-between because it is change layout

**Pull request closed without merging**